hamlet/celery.py

```diff
@@ -455,27 +468,36 @@ def ml_subject_assistant_export_to_microsoft_pt3_create_shareable_azure_blob(
     return shareable_file_url
 
 def ml_subject_assistant_export_to_microsoft_pt4_make_ml_request(shareable_file_url):
-    ml_task_id = None
+  
+    print('[Subject Assistant] Exporting to Microsoft 4/4: make request to Microsoft')
```

Comment:
Should these print statements use the logger? E.g. `logger.info(msg)`

Reply:
Yeah, using the logger would be preferable. I'm not 100% sure that `print` output will make it to the logs in production.

Reply:
Noted on using a proper logging system, but for now I think I'm going to keep `print()` - it serves its purpose and is fairly stable on my tests.

I actually tried adding `import logging ; logger.info('beep boop')` to see if it was a better option, but that crashed the Celery code so hard (apparently I forgot to instance a logger object) that it took me a two hours to fix my mistake, so I'm a bit cautious now.